addons/web/static/src/views/kanban/kanban_column_progressbar.scss

```diff
@@ -42,5 +42,6 @@
         @include o-kanban-css-filter(danger, map-get($theme-colors, "danger"));
         @include o-kanban-css-filter(muted, map-get($theme-colors, "dark"));
         @include o-kanban-css-filter(info, map-get($theme-colors, "info"));
+        @include o-kanban-css-filter(purple, purple);
```

Comment:
why `purple` ? can't you use `$o-enterprise-color` ?

Reply:
Because it's the color used :
![Capture d’écran 2023-05-05 à 15 35 04](https://user-images.githubusercontent.com/80679690/236472349-d48931a8-07f0-4755-ab34-61f6177ff200.png)

Reply:
Ok, I see... I thought that you had set `purple` for the status components as well... 
Ok for now, but this is very weird... we should use variable there (and preferably not the danger/warning/... ones)